petabtests/cases/v2.0.0/pysb/0005/_parameters.tsv

```diff
@@ -1,7 +1,7 @@
-parameterId	parameterScale	lowerBound	upperBound	nominalValue	estimate
-a0	lin	0	10	1.0	1
-b0	lin	0	10	0.0	1
-k1	lin	0	10	0.8	1
-k2	lin	0	10	0.6	1
-offset_A_c0	lin	0	10	2.0	1
-offset_A_c1	lin	0	10	3.0	1
+parameterId	lowerBound	upperBound	nominalValue	estimate	priorDistribution	priorParameters
+a0	0.0	10.0	1.0	true		
```

Comment:
Looks like unneeded `prior*` columns are automatically exported by the converter?

Reply:
That's how things are currently serialized in libpetab. Iirc there has been a request to always write all columns. No strong opinion. I'd just go with the default from libpetab here, whichever that will be.